src/snapred/backend/data/LocalDataService.py

```diff
@@ -745,7 +757,12 @@ def readCifFilePath(self, sampleId: str):
     @validate_call
     def readCalibrationState(self, runId: str, useLiteMode: bool, version: Optional[int] = None):
         if not self.calibrationExists(runId, useLiteMode):
-            raise RecoverableException.stateUninitialized(runId, useLiteMode)
+            if self._hasWritePermissionsCalibrationStateRoot():
+                raise RecoverableException.stateUninitialized(runId, useLiteMode)
+            else:
+                raise RuntimeError(
+                    "No calibration exists, and you lack permissions to create one." " Please contact your CIS."
```

Comment:
ruff seems to think this should be one line so it shall be

Reply:
if you put a `#` it could silence it